Cargo.toml

```diff
@@ -45,13 +45,21 @@ proc-mounts = { version = "0.2.4", optional = true }
 # [1]: https://github.com/rust-lang/cargo/issues/1596
 fuse = { version = "0.3.1", optional = true }
 
+[target.'cfg(target_os = "macos")'.dependencies]
+cocoa = { git = "https://github.com/servo/core-foundation-rs.git", rev = "786895643140fa0ee4f913d7b4aeb0c4626b2085", optional = true }
+objc = { version = "0.2", optional = true }
```

Comment:
These seem like a quite heavy dependencies, I wonder if there is a way to avoid these. Moreover, I would prefer not to depend on anything that is not on crates.io because of our internal build infrastructure.